vite.config.ts

```diff
@@ -17,5 +17,4 @@ export default defineConfig({
       '@': fileURLToPath(new URL('./src', import.meta.url))
     }
   },
-  base: '/frink-query-ui/'
```

Comment:
Before merging, we need to figure out how to either add this back in the GitHub deployment, or remove it in the Kubernetes deployment.

Reply:
I took care of this in #30.